src/common/transformations/include/transformations/docs/LOGGING_README.md

```diff
@@ -0,0 +1,18 @@
+# Logging of the pattern matching
+
+ The logging functionality allows to observe/debug the pattern kmatching process.
+
+In order to utilzie the logging, first, you need to set the CMake flag ```-DENABLE_OPENVINO_DEBUG=ON```
+
+_NOTE: the logging would also work if your build is configured as Release_
+
+In order to start logging, set the environmental variable ```OV_LOGGING_ENABLED``` to ``true/ON`` before running your executable or script as following:
```

Comment:
I think we should reflect in the variable name that this debug parameter is for matchers. Maybe `OV_MATCHER_LOGGING_ENABLED`?